mount.go

```diff
@@ -197,7 +197,12 @@ var storageHandlerList = map[string]storageHandler{
 }
 
 func ephemeralStorageHandler(storage pb.Storage, s *sandbox) (string, error) {
-	if _, err := os.Stat(storage.MountPoint); os.IsNotExist(err) {
+	s.Lock()
+	defer s.Unlock()
+	mountTmpfs := s.setSandboxStorage(storage.MountPoint)
```

Comment:
Use a clearer name here. I would use `existStorage` instead of `mountTmpfs`, and then for the `if` statement, I would use the negative form:
```go
if !existStorage {
...
}
```

It looks clearer to me that we're trying to create the path/storage if it does not exist.